client/components/Muncher/Muncher.js

```diff
@@ -1,15 +1,65 @@
 import { useState } from 'react';
 import { useUser } from '../../context/UserContext.js';
+import { useMuncher } from '../../hooks/useMuncher.js';
 import { enforceUser } from '../../services/UserService.js';
+import SpeechToText from '../SpeechToText/SpeechToText.js';
 import styles from './Muncher.css';
 
 export default function Muncher() {
   const { user, loading } = useUser();
   enforceUser(user, loading);
 
-  const [game, setGame] = useState();
+  const { 
+    game,  
+    targetSound, setTargetSound, 
+    points, 
+    lives, 
+    gameover,
+    resetGame,
+    currentPosition,
+    handleMove,
+    handleEat,
+  } = useMuncher();
+
+  if (gameover) {
+    return (
+      <div>
+        <h2 className={'title'} >
+          Game Over!
+        </h2>
+        <h2>total points: {points}</h2>
+        <div>
+          <button onClick={resetGame}>
+            Play again?
+          </button>
+        </div>
+      </div>
+    );
+  }
 
   return (
-    <div>Muncher</div>
+    <>
+      <h2 className={'title'}>
+        {targetSound} as in bee
```

Comment:
I notice that this title only works for the "ee" (and maybe "ea" unless the word is bread or something). Are you planning on also implementing an array of example words for each sound to populate the title?

Reply:
Yes! discussed in an above comment